src/main/java/net/minecraftforge/client/model/pipeline/BlockInfo.java

```diff
@@ -35,7 +35,7 @@
     private IBlockState state;
     private BlockPos blockPos;
 
-    private final boolean[][][] translucent = new boolean[3][3][3];
+    private final boolean[][][] t = new boolean[3][3][3];
```

Comment:
Is there any reason to rename this field?

Reply:
seems to be so the code below can nicely align in columns to be at least half-way readable